### PR TITLE
Describe usage and requirement for running Vespa container startup di…

### PR DESCRIPTION
**Pull request by ean**
…rectly as Vespa user

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

@kkraune 

@jobergum - does this make sense to you as well?

@aressem - FYI

### CODE REVIEW

**Review by kkraune — APPROVED** [review]
Looks good to me! Thanks

File: en/operations/docker-containers.html [hunk 1]
```diff
@@ -33,7 +33,32 @@ <h2 id="mounting-persistent-volumes">Mounting persistent volumes</h2>
   --volume $VESPA_LOG_STORAGE:/opt/vespa/logs --publish 8080:8080 vespaengine/vespa
 </pre>
 
+<h2 id="">Start Vespa container with Vespa user</h2>
 
+<p>
+  You can start the container directly as the `vespa` user. The `vespa` user and group within the container are
+  configured with user id `1000` and group id `1000`. The vespa user and group must be the owner of the `var` and
```
Comment (jobergum) [inline en/operations/docker-containers.html:40]: `/opt/vespa/var`  and `/opt/vespa/logs` ?

**Review by jobergum — APPROVED** [review]
Great, consider using full path names.

File: en/operations/docker-containers.html (hunk 1 above)
Comment (ean) [reply en/operations/docker-containers.html]: agree, fixed.

**Pull request merged**